doc/user-guide/pandas.rst

```diff
@@ -110,6 +110,62 @@ work even if not the hierarchical index is not a full tensor product:
     s[::2]
     s[::2].to_xarray()
 
+Lossless and reversible converter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The previous example shows that the conversion is not reversible (lossless roundtrip) and that the size of the ``datasets`` increases.
+
+Another approach is to use a lossless and reversible conversion (e.g Third party `ntv-pandas`__ libraries). A dataset can then be shared
+between several tools.
+
+__ https://github.com/loco-philippe/ntv-pandas/blob/main/README.md
+
+DataFrame to Dataset or DataArray
+---------------------------------
+
+The conversion is done without loss, by finding the multidimensional structure hidden by the tabular structure.
+
+By applying this conversion to the DataFame above, we find the initial ``Dataset``:
+
+.. ipython:: python
+
+    import ntv_pandas as npd
+
+    df.npd.to_xarray()
+
+Dataset or DataArray to Dataframe
+---------------------------------
+
+In the other direction, information that is not supported by the DataFrame must be transferred to the DataFrame (e.g. ``attrs`` data).
```

Comment:
* Can you use `ds` from above (`ds = df.npd.to_xarray()`) to show the round trip?
* This sentence sounds like `attrs` is important for the structure, but it does not seem to be.
* Maybe you can start with the back-transformation from above and just mention the `attrs` business afterwards?

Reply:
Yes you are right, `attrs` is not important for the structure. It's important only to be lossless.

I will follow your advice in the next commit.